ViewControllers/LoginPageViewController.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+
+class LoginPageViewController: UIViewController {
+    
+    //MARK: - Outlets
+    @IBOutlet weak var lbLogin: UILabel!
+    
+    //MARK: - UIViewControllers
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addGuesture()
+    }
+    
+    //MARK: - Actions
+    @IBAction func goBackToHome(_ sender: Any) {
+        dismiss(animated: true, completion: nil)
+    }
+    
+}// End of class
+
+//MARK: - Extension
```

Comment:
Write File private function as mark
Put mark according to you content inside that block